crates/astria-conductor/src/celestia/reconstruct.rs

```diff
@@ -56,7 +57,12 @@ pub(super) fn reconstruct_blocks_from_verified_blobs(
                 celestia_height,
                 block_hash: header_blob.block_hash(),
                 header: header_blob.into_unchecked().header,
-                transactions: rollup.into_unchecked().transactions,
+                transactions: rollup
+                    .into_unchecked()
+                    .transactions
+                    .into_iter()
+                    .map(Bytes::into)
+                    .collect(),
```

Comment:
Managed to get a few layers deeper, but found no other way than Vec <> Byte conversion when it came to `RollupData`, since its `SequencedData` variant is of type `Vec<Bytes>`, but its `Deposit` variant is of type `Box<Deposit>`. Additionally, due to changing the type of `RollupTransactions`, the new commit was failing this test: https://github.com/astriaorg/astria/blob/d92a37366a01d190a92f578d763ec54c8b0d15cc/crates/astria-conductor/src/sequencer/reporting.rs#L79
I generated another snapshot, but let me know if you think this is appropriate. Thanks for the feedback :)

Reply:
Looks like this is sorted now, and the snapshots remain unchanged, which is what we want :)